.storybook/decorators/language.js

```diff
@@ -17,6 +18,34 @@ export const withLanguageWrapper = makeDecorator({
 		} = context;
 
 		useEffect(() => {
+			const isNotEnglish = lang && lang !== "en-US";
+
+			// If it is US-language or unset use the rok6rmo Adobe font web project id (smaller size),
+			// otherwise use the mge7bvf kit with all the language settings (larger size)
+			const kitId = isNotEnglish ? "mge7bvf" : "rok6rmo";
+			const config = {
+				kitId,
+				async: true,
+				scriptTimeout: 3000,
+				// https://github.com/typekit/webfontloader?tab=readme-ov-file#configuration
+				loading: function() {},
+				fontactive: function(familyName) {
+					console.log(`Font ${familyName} active`);
```

Comment:
Should we remove this logging before merge? It's creating a lot of log entries. And duplicative ones for some reason:
<img width="340" alt="Screenshot 2024-07-29 at 10 11 36 AM" src="https://github.com/user-attachments/assets/0de2e25a-9180-459e-b318-132838ab5f6c">

Reply:
I do want to go back and look into why it's logging multiple times, let me add a ticket for this.

Reply:
Opened in Jira as CSS-866